project-squirrel-dragon/CoreLayer/ClientApis/PokemonTcg/PokemonTcgApiClient.swift

```diff
@@ -35,7 +35,8 @@ class PokemonTcgApiClient: PokemonTcgApiClientProtocol {
     }
 
     func fetch(itemWithID itemID: String, then handler: @escaping AnyResultBlock) {
-        endPoint.queryItems = [URLQueryItem(name: itemID, value: nil)]
+        endPoint.queryItems = [URLQueryItem(name: "q", value: itemID)]
```

Comment:
Think that this might be some temporary code?

Reply:
this is creating the query for a specific item, so its not temporary